src/Configuration/src/ConfigServerBase/ConfigurationSettingsHelper.cs

```diff
@@ -122,9 +122,9 @@ private static bool GetEnabled(IConfigurationSection clientConfigsection, bool d
         return clientConfigsection.GetValue("enabled", def);
     }
 
-    private static string GetToken(IConfigurationSection clientConfigsection)
+    private static string GetToken(IConfigurationSection clientConfigsection, string def)
```

Comment:
I'm assuming that `def` means `defaultValue`?

Reply:
Yeah. Not the best, but I was just matching the methods around it and figured we'd make sure it all looks better when bringing the change to 4.0

Reply:
I'm okay with that.